### PR TITLE
chore(server): add mina

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: server/config/deploy.rb [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+# Basic settings:
+#   domain       - The hostname to SSH to.
+#   deploy_to    - Path to deploy into.
+#   repository   - Git repo to clone from. (needed by mina/git)
+#   branch       - Branch name to deploy. (needed by mina/git)
+
+set :application_name, "ckb-explorer"
+set :domain, "47.97.171.140"
+set :deploy_to, "/home/deploy/ckb-explorer"
+set :repository, "https://github.com/shaojunda/ckb-explorer.git"
```
Comment (classicalliu) [inline server/config/deploy.rb]: fix it

File: server/config/deploy.rb [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+set :branch, "develop"
+set :rails_env, "production"
+set :user, "deploy"
+set :rvm_use_path, "/usr/share/rvm/scripts/rvm"
```
Comment (classicalliu) [inline server/config/deploy.rb]: Remove this line.

Comment (shaojunda) [reply server/config/deploy.rb]: why? without this will failed.

Comment (shaojunda) [reply server/config/deploy.rb]: image

Comment (classicalliu) [reply server/config/deploy.rb]: Move to staging file, it's not rvm default path, rvm default is `/home/username/.rvm/bin/rvm`, only installed by root will get this path and mina can't find correctly.

Comment (shaojunda) [reply server/config/deploy.rb]: ok, thank you.

**Pull request merged**